packages/zoe/src/zoeService/zoeSeat.js

```diff
@@ -154,12 +153,12 @@ export const makeZoeSeatAdminFactory = baggage => {
           // Since this method doesn't wait, we could re-enter via exitAllSeats.
           // If that happens, we shouldn't re-do any of the work.
           if (state.exiting) {
-            return;
+            return Promise.resolve();
           }
           assertHasNotExited(this, 'Cannot exit seat. Seat has already exited');
 
           state.exiting = true;
-          E.when(
+          return E.when(
```

Comment:
This conflicts with the interface pattern `.returns()`

Reply:
That was temporary debugging state. I did have to fix it, but it's been reverted.